<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Pump_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6 번 펌프</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>741</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnpower">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Power ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enstop">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E - Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
